{"name": "test_create_user", "status": "broken", "statusDetails": {"message": "FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\TT\\\\Desktop\\\\Test.json'", "trace": ">   ???\n\nC:\\Users\\TT\\PycharmProjects\\REST Automation 1\\TestCases\\test_create_user.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nfile_path = 'C:\\\\Users\\\\TT\\\\Desktop\\\\Test.json'\n\n>   ???\nE   FileNotFoundError: [Errno 2] No such file or directory: 'C:\\\\Users\\\\TT\\\\Desktop\\\\Test.json'\n\nC:\\Users\\TT\\PycharmProjects\\REST Automation 1\\TestCases\\test_create_user.py:11: FileNotFoundError"}, "start": 1608055588556, "stop": 1608055588567, "uuid": "e38f7ea6-0c9b-4524-b270-b1e26f9adee3", "historyId": "8521a9d9777323ca68be9e5019d3f66d", "testCaseId": "fe7136b806aa46c7850cd3f789474cc4", "fullName": "TestCases.test_create_user#test_create_user", "labels": [{"name": "parentSuite", "value": "TestCases"}, {"name": "suite", "value": "test_create_user"}, {"name": "host", "value": "DESKTOP-NMI0QKM"}, {"name": "thread", "value": "6852-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCases.test_create_user"}]}